pkg/ccl/backupccl/backup_test.go

```diff
@@ -216,6 +216,7 @@ func TestBackupRestorePartitioned(t *testing.T) {
 	args := base.TestClusterArgs{
 		ServerArgsPerNode: map[int]base.TestServerArgs{
 			0: {
+				DisableDefaultTestTenant: true,
```

Comment:
can we comment the reason on one of these disables, and if its something that we can overcome file a tracking issue that is mentioned in the comment?